docs/docsite/rst/dev_guide/testing_units.rst

```diff
@@ -35,18 +40,19 @@ Or against a specific Python version by doing:
    ansible-test units --tox --python 2.7 apt
 
 
-
 For advanced usage see the online help::
 
    ansible-test units --help
 
+You can also run tests in Ansible's continuous integration system by opening a pull request which will run the full test suite 
```

Comment:
You can also run tests in Ansible's continuous integration system by opening a pull request. The tests executed will be determined by the changes made in your pull request.